paper_framework_sources/p1897_initial_executor_algorithms_for_cpp23.md

```diff
@@ -501,24 +544,35 @@ float r =
       [](int a, float b){return a + b;}));
 // r==4.2
 ```
+*- end example]*
 
 ### Wording
-
 The name `execution::when_all` denotes a customization point object.
-The expression `execution::when_all(S)` for some subexpression `S` is expression-equivalent to:
+For some subexpression `ss`, let `Ss` be a list of types such that `decltype((ss))...` is `Ss...`.
+The expression `execution::when_all(ss...)` is expression-equivalent to:
 
- * Otherwise, `when_all(S)` if that expression is valid with overload resolution performed in a context that includes the declaration
+ * `when_all(ss...)` if that expression is valid, and if `S` satisfies `sender` and if `Sch` satisfies `scheduler`, with overload resolution performed in a context that includes the declaration
```

Comment:
I do not see where Sch comes from. I would expect Sch to come from the output_receiver if it is a scheduler_provider.

Reply:
Yes, that's broken copy/paste text.